### ISSUE TITLE
'null' user after importing plexWatch db

### ISSUE THREAD

**Issue by scottjasso** [Tautulli/Tautulli#350]
For some reason, after importing my plexWatch db, the history page shows my own accounts views as user 'null'. Nobody else's account is affected. Clicking on 'null' loads "/plexpy/user?user_id=1", and it has my actual username (wscottj5), not null. This page has all my view history from the plexWatch db. This 'null' user does not show up on the users page.

This only affects imported data - any new shows I watch show up properly in history, with user wscottj5 and user_id 3486746. However this user is missing all my history (which is back in user_id 1). 

Imported from plexWatch v0.3.1.

**Comment by drzoidberg33** [comment]
Have you changed Plex accounts during the time between the PlexWatch data was first captured and now?

**Comment by scottjasso** [comment]
No, I don't think so. The only related change I can think of is adding a
guest account, and I don't think that would affect anything.

**Comment by drzoidberg33** [comment]
Thanks, I've replicated this issue on my end. Seems to be a regression in the last update.

Working on a fix.

**Comment by drzoidberg33** [comment]
@wjohnson5 

This issue has been resolved in v1.2.14. If you don't wish to start from scratch, you can download a sqlite editor (like sqlite browser) and run the following command on the plexpy.db file (just ensure you shut down PlexPy first).

```
UPDATE session_history SET user_id = 3486746 WHERE user_id = 1
```

Then make sure you write the changes and exit sqlite browser. Then restart PlexPy.

**Comment by scottjasso** [comment]
Thanks! I haven't tested the new version, but the sql query worked fine. Thanks for the quick turnaround!
